src/main/java/com/rapid7/appspider/ScanManagement.java

```diff
@@ -1,10 +1,10 @@
 package com.rapid7.appspider;
 
+import com.google.inject.internal.guava.base.$Optional;
 import org.json.JSONObject;
 import org.json.JSONArray;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
In general, avoid * imports, only take what you need

Reply:
Intellij auto imported it, guess it prefers .*  I'll remove it and see what complains

Reply:
updated to be specific, found another in PostBuildScan which was updated as well